src/us/kbase/workspace/test/database/provenance/RelationshipTypeTest.java

```diff
@@ -23,24 +23,30 @@ public void testGetPid() throws Exception {
 		assertThat("incorrect PID",
 				rt2.getPid(),
 				is("DataCite:IsPreviousVersionOf"));
+		final RelationshipType rt3 = RelationshipType.IS_MANIFESTATION_OF;
+		assertThat("incorrect PID",
+				rt3.getPid(),
+				is("Crossref:IsManifestationOf"));
 	}
 
 	@Test
```

Comment:
It'd be good to have a test that goes from a string -> non-duplicate crossref type

Reply:
I meant the other way, using the string -> enum map

Reply:
Not that this is bad

Reply:
:+1: